<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTDDWidget</class>
 <widget class="QWidget" name="MainTDDWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Treatment Delivery Device Emulator</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="group_box_aes_and_machine">
     <property name="title">
      <string>Remote System Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5">
       <widget class="QLineEdit" name="machine_name_line_edit"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="machine_name_label">
        <property name="text">
         <string>Machine Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="qr_ae_label">
        <property name="text">
         <string>QR AE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="qr_ae_line_edit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ups_ae_label">
        <property name="text">
         <string>UPS AE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ups_ae_line_edit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="group_box_tdd_scp">
     <property name="title">
      <string>TDD Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QPushButton" name="import_staging_directory_push_button">
        <property name="text">
         <string>Choose Staging Directory</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QCheckBox" name="auto_download_checkbox">
        <property name="text">
         <string>Auto Download</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QLineEdit" name="import_staging_dir_line_edit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="tdd_ae_label">
        <property name="text">
         <string>TDD Event and Store SCP AE</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Import Staging Directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="tdd_ae_line_edit"/>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="subscribe_ups_checkbox">
        <property name="text">
         <string>Subscribe to UPS</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="tdd_restart_push_button">
        <property name="text">
         <string>Restart SCP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Reference Data Retrieval</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="get_listed_inputs_push_button">
        <property name="toolTip">
         <string extracomment="Will retrieve from all UPS">Will retrieve from all UPS</string>
        </property>
        <property name="whatsThis">
         <string extracomment="Button to get all references from Input Information Sequence">Button to get all references from Input Information Sequence</string>
        </property>
        <property name="text">
         <string>Get Listed Inputs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="get_rtss_and_ct_push_button">
        <property name="toolTip">
         <string>Will retrieve from all UPS</string>
        </property>
        <property name="text">
         <string>Get RTSS and CT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Worklist Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="horizontalSpacing">
       <number>-1</number>
      </property>
      <property name="verticalSpacing">
       <number>-1</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="group_box_cfind_request_and_response">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="0">
          <widget class="QDateTimeEdit" name="latest_date_time_edit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="push_button_get_ups">
           <property name="text">
            <string>Get UPS</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDateTimeEdit" name="soonest_date_time_edit">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="scheduled_date_label">
           <property name="text">
            <string>scheduled date range</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="step_status_label">
           <property name="text">
            <string>Procedure Step Status</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="schedule_to_label">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="step_status_combo_box">
           <property name="maxVisibleItems">
            <number>6</number>
           </property>
           <property name="maxCount">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QScrollArea" name="scrollArea">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>706</width>
           <height>177</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTreeWidget" name="ups_response_tree_widget">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">Tag</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>VR</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Keyword</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Session Progress</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="in_progress_button">
        <property name="text">
         <string>Start Procedure</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="transaction_uid_line_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="beam_number_label">
        <property name="text">
         <string>Beam Number</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="beam_number_spin_box">
        <property name="maximumSize">
         <size>
          <width>758</width>
          <height>21</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="session_percent_label">
        <property name="text">
         <string>Session Percent Complete</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="session_percent_spin_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="update_progress_button">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="final_update_button">
        <property name="text">
         <string>Finish</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="send_tx_record_button">
        <property name="text">
         <string>Send Tx Record</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="cancel_procedure_button">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="complete_procedure_button">
        <property name="text">
         <string>Complete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
